backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -182,7 +182,7 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
     val rcPath = cwd./(jobPaths.returnCodeFilename)
     val rcTmpPath = rcPath.plusExt("tmp")
 
-    val globFiles = backendEngineFunctions.findGlobOutputs(call, jobDescriptor)
+    val globFiles: Set[WdlGlobFile] = call.globFiles(jobDescriptor.fullyQualifiedInputs)
```

Comment:
ToL: I think I lied and that we *can* (doesn't mean we have to) do this with what we have already:

```scala
def findGlobOutputs(call: TaskCallNode, jobDescriptor: BackendJobDescriptor): ErrorOr[List[WdlGlobFile]] = {
    call.callable.outputs.flatTraverse[ErrorOr, WdlGlobFile] {
      _.expression.evaluateFiles(jobDescriptor.fullyQualifiedInputs, this, WdlAnyType) map {
        _.toList collect { case glob: WdlGlobFile => glob }
      }
    }
  }
```

Reply:
By all means, let's use what we already have!  Does this fall in `TaskDefinition`?  I.e. what is `this`?

Reply:
It falls in the `GlobFunctions` trait. `this` is the `IoFunctionSet`. WDL needs it (or at least claims it does...) to find out what the globs are.